cli/src/cli.rs

```diff
@@ -89,6 +89,20 @@ pub(crate) enum Command {
         #[arg(short, long, action = ArgAction::SetTrue, conflicts_with_all = ["from_private_key", "zero"], required_unless_present_any = ["from_private_key", "zero"], help = "A random address.")]
         random: bool,
     },
+    #[clap(
+        about = "Get either the keccak for a given input, the zero hash, the empty string, or a random hash",
+        visible_alias = "h"
+    )]
+    Hash {
+        #[arg(long, value_parser = parse_hex, conflicts_with_all = ["zero", "random", "string"], required_unless_present_any = ["zero", "random", "string"], help = "The input to hash.")]
+        input: Option<Bytes>,
+        #[arg(short, long, action = ArgAction::SetTrue, conflicts_with_all = ["input", "random", "string"], required_unless_present_any = ["input", "random", "string"], help = "The zero hash.")]
+        zero: bool,
+        #[arg(short, long, action = ArgAction::SetTrue, conflicts_with_all = ["input", "zero", "string"], required_unless_present_any = ["input", "zero", "string"], help = "A random hash.")]
+        random: bool,
+        #[arg(short, long, action = ArgAction::SetTrue, conflicts_with_all = ["input", "zero", "random"], required_unless_present_any = ["input", "zero", "random"], help = "Hash of empty string")]
```

Comment:
Nit: I believe `clap` supports using `enum` for this, which would make this incantation more brief and convert the latter reference to `Command::Hash` into a simple `match`.